scripts/gulpfiles/build_tasks.js

```diff
@@ -97,48 +100,48 @@ const chunks = [
     name: 'blockly',
     entry: path.join(TSC_OUTPUT_DIR, 'core', 'main.js'),
     exports: 'module$build$src$core$blockly',
-    reexport: 'Blockly',
+    scriptExport: 'Blockly',
   },
   {
     name: 'blocks',
     entry: path.join(TSC_OUTPUT_DIR, 'blocks', 'blocks.js'),
     exports: 'module$exports$Blockly$libraryBlocks',
-    reexport: 'Blockly.libraryBlocks',
+    scriptExport: 'Blockly.libraryBlocks',
   },
   {
     name: 'javascript',
     entry: path.join(TSC_OUTPUT_DIR, 'generators', 'javascript', 'all.js'),
     exports: 'module$build$src$generators$javascript$all',
-    reexport: 'Blockly.JavaScript',
-    reexportOnly: 'javascriptGenerator',
+    scriptExport: 'javascript',
+    scriptNamedExports: {'Blockly.Javascript': 'javascriptGenerator'},
```

Comment:
As @maribethb noted: this should have been `Blockly.JavaScript`, not `…Javascript`.